server_tests/test/clientTests.js

```diff
@@ -687,39 +686,39 @@ describe("RUST API TEST SUITE", function() {
             this.timeout(timeoutValue);
             return testClient.takePicture(sessionId)
             .then(function onSuccess (res) {
-                Comparison.oscTakePictureOutput(res);
+                validate.done(res.body, schema.names.commandTakePicture);
                 fileUri = res.body.results.fileUri;
                 return testClient.getImage(fileUri);
             })
-            .then((res) => Comparison.oscGetImageOutput(res))
+            .then((res) => validate.done(res.body, schema.names.commandGetImage))
             .catch(wrapError);
         });
 
         it("Expect success. camera.getImage successfully gets image when provided with a valid fileUri and maxSize", function() {
             this.timeout(timeoutValue);
             return testClient.takePicture(sessionId)
             .then( function onSuccess (res) {
-                Comparison.oscTakePictureOutput(res);
+                validate.done(res.body, schema.names.commandTakePicture);
                 fileUri = res.body.results.fileUri;
                 return testClient.getImage(fileUri, 100);
             })
             .then( function onSuccess (res) {
-                Comparison.oscGetImageOutput(res);
+                validate.done(res.body, schema.names.commandGetImage);
```

Comment:
But for the missingParameterErrors and invalidParameterValueErrors the schema test is still sufficient right? So, no need to modify those?